lib/boavizta.ts

```diff
@@ -0,0 +1,135 @@
+import {IImpactModelInterface, IStaticParams} from "./interfaces/index";
+import axios from "axios";
+
+export {
+    IImpactModelInterface,
+    IStaticParams
+} from "./interfaces/index";
+
+
+export interface IBoaviztaStaticParams extends IStaticParams {
```

Comment:
Remove

Reply:
It will not be possible use "in" without casting.